sessions/css-positioning/demo-end/index.html

```diff
@@ -6,215 +6,119 @@
     />
-    <title>HTML & CSS</title>
+    <title>CSS Positioning</title>
     <link rel="stylesheet" href="./css/styles.css" />
   </head>
   <body>
     <header class="header">
-      <h1 class="header__heading">CSS positioning</h1>
+      <h1 class="header__heading">
+        Sweeties<span class="plate" role="img" aria-label="A plate">🍽</span>
+      </h1>
     </header>
-    <main>
-      <section class="disclaimer">
-        <h2 class="disclaimer__title">disclaimer</h2>
-        <div class="disclaimer__icon">!</div>
-        <p class="disclaimer__text">
-          Lorem ipsum dolor sit, amet consectetur adipisicing elit. Alias vitae
-          libero harum! Odit consequatur veritatis, hic qui, facere distincLorem
-          ipsum dolor sit, veritatis, hic qui, facere distincorem ipsum dolor
-          sit, amet consectetur adipisicing elit. Alias vitae libero harum! Odit
-          consequatur veritatis, hic qui, facere distinc
+
+    <main class="main">
+      <section class="main__section">
+        <p class="main__text">
+          From hard candies to gummies, chocolates to caramels, we've got
+          something for everyone to satisfy their sweet tooth!
+        </p>
+        <span class="donut" role="img" aria-label="A chocolate donut">🍩</span>
+        <span class="donut-text">Delicious!</span>
+        <p class="main__text">
+          First, our hard candies come in a variety of flavors, including
+          classic favorites like cherry, lemon, and lime, as well as unique
+          blends like watermelon-mint and strawberry-basil. They're perfect for
+          enjoying slowly, letting the flavor melt and linger in your mouth.
+        </p>
+        <p class="main__text">
+          For those who prefer a chewier candy, our gummies are a must-try.
+          We've got everything from classic gummy bears to sour worms and fruit
+          slices, with flavors like grapefruit, peach, and raspberry. Our
+          gummies are soft and chewy, bursting with fruity goodness.
+        </p>
+        <p class="main__text">
+          For the chocolate lovers out there, Sweeties has you covered with our
+          delicious chocolate creations. From rich, creamy truffles to
+          chocolate-covered pretzels, there's a treat to suit every taste. We
+          use only the finest ingredients, including premium cocoa and fresh
+          nuts, to create our chocolates.
+        </p>
+        <span class="separator" role="img" aria-label="A piece of hard candy"
+          >🍬</span
+        >
+        <p class="main__text">
+          Our caramels are some of the best around. We've got classic flavors
+          like vanilla and chocolate, as well as unique twists like sea salt and
+          honey. Our caramels are soft and buttery, with just the right amount
+          of sweetness.
+        </p>
+        <p class="main__text info">
+          <span
+            class="info-accent"
```

Comment:
```suggestion
            class="content__info--accent"
```

Reply:
The little icon here isn't a variant of `content__info` 🤔 .

Reply:
You are right, I was distracted by the info part. 
info__accent would be the way to go I guess. one `-` is not sufficient either way :)

Reply:
BEM is hard.. I don't actually know if you have to stay consistent with your Blocks, or if you can redefine the Blocks now and then. Because content__text--info makes sense to me in the larger context, but here the info you be the new block and referring to the same thing